openlibrary/plugins/upstream/forms.py

```diff
@@ -75,7 +75,7 @@ class RegisterForm(Form):
     INPUTS = [
         Textbox(
             'email',
-            description=_('Your email address'),
+            description=_('Email Address *'),
```

Comment:
I'm not sure if this asterisk should always be displayed.  I think that it's meant to indicate that validation has failed.
Have raised these questions [here](https://github.com/internetarchive/openlibrary/issues/7644#issuecomment-1624178140).

Reply:
This will not be resolved until the asterisks and validation questions are answered.